### PR TITLE
PENG-252 :: Add editable code to the NPM library

### PR DESCRIPTION
**Pull request by kevin-uzcategui**
The component that is in the ui-library called EditableCode is already working in StoryBook, but the necessary configurations will be made so that it works with the NPM package so that it can be used in workflow-web

### CODE REVIEW

File: .github/workflows/publish-package.yml [hunk 1]
```diff
@@ -15,6 +15,14 @@ jobs:
...
           registry-url: 'https://registry.npmjs.org'
       - run: npm install
+      - run: npm install storybook
+      - run: npm install @storybook/addon-links
+      - run: npm install @storybook/addon-onboarding
+      - run: npm install @storybook/addon-actions
+      - run: npm install @storybook/addon-controls
+      - run: npm install @storybook/addon-docs
+      - run: npm install @storybook/addon-essentials
+      - run: npm install @storybook/addon-storysource
```
Comment (jaristizabal-fuse) [inline .github/workflows/publish-package.yml:25]: Why did you move them from the package.json to here?

**Review by jaristizabal-fuse — COMMENTED** [review]
Please check my questions.

File: .github/workflows/publish-package.yml (hunk 1 above)
Comment (kevin-uzcategui) [reply .github/workflows/publish-package.yml:25]: No, the storybook library has conflicts with workflow-web and TypeScript.

That's why I separated it.

Comment (jaristizabal-fuse) [reply .github/workflows/publish-package.yml:25]: Okay Good!

**Review by jaristizabal-fuse — APPROVED** [review]
